<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::PrivacyPreferencesPage</class>
 <widget class="QWidget" name="Otter::PrivacyPreferencesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="trackingHeader">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tracking</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="doNotTrackLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="doNotTrackLabel">
       <property name="text">
        <string>Do Not Track:</string>
       </property>
       <property name="buddy">
        <cstring>doNotTrackComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="doNotTrackComboBox">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="historyHeader">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>History</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="privateModeCheckBox">
     <property name="text">
      <string>Private mode</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="historyLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="historySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="historyWidget" native="true">
       <layout class="QVBoxLayout" name="historyWidgetLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="rememberBrowsingHistoryCheckBox">
          <property name="text">
           <string>Remember browsing history</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="rememberDownloadsHistoryCheckBox">
          <property name="text">
           <string>Remember downloads history</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="rememberSearchHistoryCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remember search history</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="rememberFormsHistorylLayout">
          <item>
           <widget class="QCheckBox" name="rememberFormsHistoryCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remember form history</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="rememberFormsHistorySpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="rememberFormsHistoryButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Template…</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="enableCokiesLayout">
          <item>
           <widget class="QCheckBox" name="enableCookiesCheckBox">
            <property name="text">
             <string>Enable cookies</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="cookiesDetailsLayout">
          <item>
           <spacer name="cookiesDetailsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="cookiesWidget" native="true">
            <layout class="QFormLayout" name="cookiesLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="cookiesPolicyLabel">
               <property name="text">
                <string>Accept cookies:</string>
               </property>
               <property name="buddy">
                <cstring>cookiesPolicyComboBox</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cookiesPolicyComboBox"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="keepCookiesModeLabel">
               <property name="text">
                <string>Keep until:</string>
               </property>
               <property name="buddy">
                <cstring>keepCookiesModeComboBox</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="keepCookiesModeComboBox"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="thirdPartyCookiesPolicyLabel">
               <property name="text">
                <string>Accept third-party cookies:</string>
               </property>
               <property name="buddy">
                <cstring>thirdPartyCookiesPolicyComboBox</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="thirdPartyCookiesLayout">
               <item>
                <widget class="QComboBox" name="thirdPartyCookiesPolicyComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="thirdPartyCookiesExceptionsButton">
                 <property name="text">
                  <string>Exceptions…</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="clearHistoryLayout">
          <item>
           <widget class="QCheckBox" name="clearHistoryCheckBox">
            <property name="text">
             <string>Clear history when application closes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="clearHistorySpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="clearHistoryButton">
            <property name="text">
             <string>Settings…</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="passwordsHeader">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Passwords</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rememberPasswordsLayout">
     <item>
      <widget class="QCheckBox" name="rememberPasswordsCheckBox">
       <property name="text">
        <string>Remember passwords</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rememberPasswordsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="managePasswordsButton">
       <property name="text">
        <string>Manage…</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="masterPasswordlLayout">
     <item>
      <widget class="QCheckBox" name="masterPasswordCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use a master password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="masterPasswordSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="masterPasswordButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Change…</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>doNotTrackComboBox</tabstop>
  <tabstop>privateModeCheckBox</tabstop>
  <tabstop>rememberBrowsingHistoryCheckBox</tabstop>
  <tabstop>rememberDownloadsHistoryCheckBox</tabstop>
  <tabstop>rememberSearchHistoryCheckBox</tabstop>
  <tabstop>rememberFormsHistoryCheckBox</tabstop>
  <tabstop>rememberFormsHistoryButton</tabstop>
  <tabstop>enableCookiesCheckBox</tabstop>
  <tabstop>cookiesPolicyComboBox</tabstop>
  <tabstop>keepCookiesModeComboBox</tabstop>
  <tabstop>thirdPartyCookiesPolicyComboBox</tabstop>
  <tabstop>thirdPartyCookiesExceptionsButton</tabstop>
  <tabstop>clearHistoryCheckBox</tabstop>
  <tabstop>clearHistoryButton</tabstop>
  <tabstop>rememberPasswordsCheckBox</tabstop>
  <tabstop>managePasswordsButton</tabstop>
  <tabstop>masterPasswordCheckBox</tabstop>
  <tabstop>masterPasswordButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
